Source/WebCore/PAL/pal/spi/mac/NSSpellCheckerSPI.h

```diff
@@ -45,6 +44,8 @@ extern NSString *NSTextCheckingSuppressInitialCapitalizationKey;
 
 @end
 
+#define _NSTextCheckingTypeSingleCompletion (1ULL << 29)
```

Comment:
Value should be redeclared matching the private header, `#define` is only for staging changes.

So:

```
typedef NS_OPTIONS(uint64_t, NSTextCheckingTypeAppKitTemporary) {
    _NSTextCheckingTypeSingleCompletion             = 1ULL << 29,
};
```

Reply:
ah, my bad, sorry!